setuptools_cmmi/process_cmmi.py

```diff
@@ -41,7 +41,7 @@ def _run(root_path, exec_name, description, args):
                                   .format(exec_path, rc))
 
 
-def process_cmmi(dest_dir, temp_source_dir, config_options, autogen, config_name=None):
+def process_cmmi(dest_dir, temp_source_dir, config_options, autogen, config_name=None, make_name=None):
```

Comment:
can we call this makefile_name? or makefile_path? make_name doesn't really describe that it's the makefile.